.github/SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Security Policy
+
+If you discover any security related issues, please email pushpak1300@gmail.com instead of using the issue tracker.
```

Comment:
Please add kite zerodha email in case of security issue

Reply:
Users can report security vulns to `bugbounty@zerodha.com `